migrating/checklists/ossm-migrating-read-me-assembly.adoc

```diff
@@ -43,6 +43,12 @@ Be aware that setting the `updateStrategy` to `RevisionBased` also has implicati
 
 include::modules/ossm-migrating-read-me-supported-multi-cluster-topologies.adoc[leveloffset=+1]
 include::modules/ossm-migrating-read-me-support-for-istioctl.adoc[leveloffset=+1]
+
+.Next steps
+You can install the Istioctl tool.
+
+* xref:../../install/ossm-istioctl-tool.adoc#ossm-installing-the-istioctl-tool_ossm-istioctl-tool[Installing the Istioctl tool]
```

Comment:
I would take advantage of this being in an assembly :)
```suggestion
.Next steps
* You can install the xref:../../install/ossm-istioctl-tool.adoc#ossm-installing-the-istioctl-tool_ossm-istioctl-tool[Istioctl tool].
```

Reply:
done